lisa/executable.py

```diff
@@ -154,6 +154,13 @@ def _check_exists(self) -> bool:
         result = self.node.execute(
             f"{where_command} {self.command}", shell=True, no_info_log=True
         )
+        if 0 != result.exit_code:
```

Comment:
What's the reason to add this logic?

Reply:
I will have a discussion with you for this.